<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerControlWidget</class>
 <widget class="QWidget" name="PlayerControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>148</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fpsLabel">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="fpsSlider">
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="fpsSpinBox">
       <property name="suffix">
        <string> fps</string>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="speedLabel">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="speedSlider">
       <property name="minimum">
        <number>-30</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="speedSpinBox">
       <property name="prefix">
        <string>1e</string>
       </property>
       <property name="minimum">
        <double>-3.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="precisionLabel">
       <property name="text">
        <string>Precision</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="precisionSlider">
       <property name="minimum">
        <number>-30</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="precisionSpinBox">
       <property name="prefix">
        <string>1e</string>
       </property>
       <property name="minimum">
        <double>-3.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="fpsDisplayLabel">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>fpsDisplayLabel</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="speedDisplayLabel">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>speedDisplayLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="precisionDisplayLabel">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string>precisionDisplayLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
